### ISSUE TITLE
multiple network requests / wildcard network request fail

### ISSUE THREAD

**Issue by bch0w** [uafgeotools/pysep#45]
From Yuan and Carl

Network selection:
- works: TA,AK,IU
- works: XV
- fails: TA,AK,IU,XV
- fails: *

By "fails" I mean that we get a subset of stations (both XV and AK) but not the full set of stations listed as having data.

The station wildcard (*) is A-OK.

The kludge is to run pysep twice, using the two "works" options for the network.

Yuan, please attach your yaml file here so that we can come back to this issue.

Bottom line: It's really important to ensure that you have all the waveforms that you are expecting!

Task List
- [ ] Figure out why multiple networks leads to subsets of stations
- [ ] Return debug log from IRIS bulk request 
- [ ] Log message at the end of PySEP telling the user if any data is NOT retrieved from their expected request.

May be related to #33
